CONTRIBUTING.md

```diff
@@ -1,5 +1,5 @@
 # Contributing
 
-For contributing to this project, please refer to the [PyAnsys Developer's Guide].
+For contributing to this project, please refer to the [PyRocky Developer's Guide].
```

Comment:
```suggestion
For information on contributing to this project, see the [PyRocky Developer's Guide].
```
Avoid "please" in technical documentation. I'm confused by the link. Doesn't this URL ( https://rocky.docs.pyansys.com/version/dev/contributing.html) refer to this the HTML output for this MD file? See this [Contribute](https://visualization-interface.tools.docs.pyansys.com/version/stable/contributing.html) section (which is the standard heading) for the introductory text normally used.

Reply:
The real "Contribution Guide" is at `doc/source/contributing.rst`. But it's a common practice for open source projects to have a `CONTRIBUTING` file at the root folder (and most PyAnsys projects follow this). So here we just point to the real "Contribution Guide".